addons/rook/template/base/operator/kustomization.yaml

```diff
@@ -4,3 +4,4 @@ resources:
 
 patchesStrategicMerge:
 - patches/deployment-tolerations.yaml
+- patches/crds.yaml
```

Comment:
Added this change

Reply:
removed in favor of following the documented steps here

https://rook.io/docs/rook/v1.6/ceph-upgrade.html#1-update-common-resources-and-crds

```
kubectl replace -f crds.yaml
kubectl apply -f crds.yaml
```